### PR TITLE
po: Add Tamil translation

### PR DESCRIPTION
**Pull request by kbdharun**
Adding Tamil translation to `po`.

### CODE REVIEW

File: po/ta.po [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.1.1\n"
+
+#. TRANSLATORS: Don't translate this text
+#: data/com.github.tchx84.Flatseal.desktop.in:4
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:4
+msgid "Flatseal"
+msgstr "பிளாட்சீல்"
```
Comment (tchx84) [inline po/ta.po]: Please check these `TRANSLATORS: Don't translate this text` messages :pray:

File: src/widgets/aboutDialog.ui [hunk 2]
```diff
@@ -28,7 +28,8 @@ Eric Zhang
 Georgi Georgiev
 Sabri Ünal
 Mathieu Bousquet
-Yaron Shahrabani</property>
+Yaron Shahrabani
+K.B.Dharun Krishna</property>
```
Comment (tchx84) [inline src/widgets/aboutDialog.ui:32]: re. commit message, can you change it to `aboutDialog: Add @kbdharun to credits`?

**Review by tchx84 — CHANGES_REQUESTED** [review]

File: po/ta.po (hunk 1 above)
Comment (kbdharun) [reply po/ta.po]: Oops, Sorry I will update it.

**Comment by kbdharun** [comment]
~Hey @tchx84 , I used `git rebase -i HEAD~7` and edited the commit. Seems like multiple commits were added can you look into my fork and remove it. Keeping the necessary ones alone.~

I will close this PR and I will open a new one with the requested changes.

**Pull request closed without merging**